malariagen_data/anoph/gplt_params.py

```diff
@@ -75,15 +75,15 @@
     "Plot title. If True, a title may be automatically generated.",
 ]
 
-figure: TypeAlias = Annotated[
+optional_figure: TypeAlias = Annotated[
     # Use quite a broad type here to accommodate both single-panel figures
     # created via bokeh.plotting and multi-panel figures created via
     # bokeh.layouts.
     Optional[bokeh.model.Model],
```

Comment:
```suggestion
    Optional[figure],
```

Small suggestion, would also require moving this below the `figure` declaration I believe.